src/routing.rs

```diff
@@ -1,30 +1,58 @@
-use axum::extract::Path;
-use axum::Router;
-use axum::routing::get;
+use std::sync::{Arc, RwLock};
```

Comment:
We are using tokio, so the synchronization primitives from tokio must be used
```suggestion
use std::sync::Arc;
use tokio::sync::RwLock;
```